scipy/stats/_multivariate.py

```diff
@@ -179,14 +179,13 @@ def __init__(self, M, cond=None, rcond=None, lower=True,
 
         # Initialize attributes to be lazily computed.
         self._pinv = None
-        self._proj = None
 
     def in_support(self, x):
         """
         Check whether x lies in the support of the distribution.
         """
-        residual = np.dot(x, self.proj.T)
-        out_of_bounds = np.any(np.abs(residual) > self.eps, axis=1)
+        residual = np.linalg.norm(x @ self.V, axis=-1)
+        out_of_bounds = residual > self.eps
         return out_of_bounds
```

Comment:
This is returning the opposite of what's intended, I believe. You should return either
```python
return ~ out_of_bounds
```
or
 ```python
in_bounds = residual < self.eps
return in_bounds
```

Reply:
Looks like the function was named opposite to that convention, and I didn't change it, but good idea - I will. Thanks!